packages/sirius-web/backend/sirius-web-starter/src/main/java/org/eclipse/sirius/web/starter/SiriusWebStarterConfiguration.java

```diff
@@ -131,12 +141,22 @@ public IEventProcessorSubscriptionProvider eventProcessorSubscriptionProvider(IE
             public Flux<IPayload> getSubscription(String editingContextId, IRepresentationConfiguration representationConfiguration, IInput input) {
                 return editingContextEventProcessorRegistry.getOrCreateEditingContextEventProcessor(editingContextId)
                         .flatMap(processor -> processor.acquireRepresentationEventProcessor(representationConfiguration, input))
-                        .map(representationEventProcessor -> representationEventProcessor.getOutputEvents(input))
+                        .map(representationEventProcessor -> customizeFlux(editingContextId, representationConfiguration, input, representationEventProcessor))
                         .orElse(Flux.empty());
             }
         };
     }
 
+    private Flux<IPayload> customizeFlux(String editingContextId, IRepresentationConfiguration representationConfiguration, IInput input, IRepresentationEventProcessor representationEventProcessor) {
+        Flux<IPayload> flux = representationEventProcessor.getOutputEvents(input);
+        for (IRepresentationEventProcessorFluxCustomizer representationEventProcessorFluxCustomizer : this.representationEventProcessorFluxCustomizers) {
+            if (representationEventProcessorFluxCustomizer.canHandle(representationConfiguration)) {
+                flux = representationEventProcessorFluxCustomizer.customize(editingContextId, representationEventProcessor, flux);
```

Comment:
I'll just align the parameters available between `canHandle` and `customize` since it can be frustrating to have access to something in one method but not in the other one